request.js

```diff
@@ -74,14 +74,14 @@
             request.onload = function(){
                 if((request.statusText === 'OK' && request.status === 200) || typeof request.statusText === 'undefined'){
                     methods.success.apply(methods, utils.parse(request));
-                    methods.always.apply();    
+                    methods.always.apply();
                 } else {
                     methods.error.apply(methods, utils.parse(request));
                     methods.always.apply();
                 }
             };
-            request.onerror = function(){
-                methods.error.apply(methods, utils.parse(request));
+            request.onerror = function(e){
+                methods.error.apply(methods, e);
```

Comment:
Not a blocker: Wouldn't be better to send the `request` as a first parameter of the `apply(...)`? It would give "more context" than a literal object (like `methods`). Just a suggestion.